setup.cfg

```diff
@@ -48,6 +48,7 @@ test =
     pytest>=7.2
     pytest-cov>=2.12,<3
     pytest-xdist>=2.3.0,<3
+    colorlog>=6.7.0
```

Comment:
It's still failing in my local setup :/

I moved it to the `install_requires` category and it seems to be working fine.